resources/aws/routetable.go

```diff
@@ -10,14 +10,14 @@ import (
 )
 
 type RouteTable struct {
```

Comment:
OK, I asked around and was told there was no agreement over this. I'll change it for this file and [created an issue](https://github.com/giantswarm/aws-operator/issues/124)